base/deprecated.jl

```diff
@@ -709,8 +709,12 @@ module Collections
         @eval Base.@deprecate_moved $f "DataStructures"
     end
 end
+@doc """
+Collections will be removed from Base. Use DataStructures instead.
```

Comment:
This is part of the 0.6 deprecation, so will never see the light of day.

Reply:
See below.